webapp/src/main/resources/properties/en.properties

```diff
@@ -285,6 +285,22 @@ workbench.toolbar.clearAllInPage=Clear all in page
 # Label displayed in the workbench toolbar dropdown to clear selection of all textunits across all pages in the search results
 workbench.toolbar.clearAll=Clear all
 
+# Title of the modal to share the workbench's search paramters
+workbench.shareSearchParamsModal.title=Share the search parameters
```

Comment:
Re naming: the term parameters might not mean anything for non-technical users? (other options: Share the search query / Share the search criteria / Share existing search etc.)

Reply:
https://github.com/pinterest/mojito/compare/700cb4a4f47f02213cfe908b20e456f68b0838c2..a1ae281c6ae7e4dd9c4e132becc8b7e3bd5ab0fe